src/lib/eliom_cscache.eliom

```diff
@@ -59,6 +78,10 @@ let%client find cache get_data id =
   try Hashtbl.find ((Eliom_shared.Value.local cache) ()) id
   with Not_found -> load cache get_data id
 
+let find_list cache get_data ids =
+  let%lwt () = cache_list cache get_data ids in
+  Lwt_list.map_s (find cache (fun _ -> raise Not_found)) ids
```

Comment:
`assert false` instead of `raise Not_found`?

Raising `Not_found` here just does not make sense.